include/taskolib/Sequence.h

```diff
@@ -313,6 +319,23 @@ class Sequence
      */
     const std::string& get_step_setup_script() const noexcept{ return step_setup_script_; }
 
+    /**
+     * Returns time of last execution. Will return Epoch time of January 1st 1970 on
+     * a fresh created sequence.
```

Comment:
Unfortunately the exact date is system-dependent. Maybe we should specify "returns TimePoint{}" for a newly created sequence.